botocove/cove_decorator.py

```diff
@@ -47,16 +48,32 @@ def _get_cove_session(
             account_details = {
                 "Id": account_id,
                 "RoleSessionName": role_session_name,
+                "Policy": session_policy,
             }
 
     else:
-        account_details = {"Id": account_id, "RoleSessionName": role_session_name}
+        account_details = {
+            "Id": account_id,
+            "RoleSessionName": role_session_name,
+            "Policy": session_policy,
```

Comment:
The RoleSessionName and the Policy details are not included in the result when calling using org_master mode.

@connelldave , is that deliberate?

It surprised me when I was testing this in my personal org.

Reply:
I think that's just a defect :) Feel free to tag it on this change if you don't mind fixing?

Reply:
Sure, I'll see what I can do!